juniper_official/Solutions/EVPN-VXLAN/spine-evpn-vxlan-kpis.playbook

```diff
@@ -0,0 +1,74 @@
+ * 7) Rule vtep-interface-status.rule, detects the vtep interface status
+ *    notify anomalies.
+ * 8) Rule vtep-interface-traffic.rule,monitors interface traffic and notifies
+      anomalies when traffic exceeds threshold.
+ * 9) Rule vtep-interface-flap.rule, detects interface flaps and notifies
+ *    anomalies when link flap count increases.
+ * 10) Rule vtep-interface-error.rule, detects interface errors and notifies
+ *     anomalies when error count increases.
+ * 11) Rule vtep-tunnel-statistics.rule, detects logical interface statisticsr
+ *     and notifies anomalies when error count increases.
+ * 12) Rule irb-interface-status.rule, detects the irb interface status
+ *     notify anomalies.
+ * 13) Rule irb-interface-traffic.rule,monitors interface traffic and notifies
+ *     anomalies when traffic exceeds threshold.
+ * 14) Rule irb-next-hops.rule, monitors irb interface next-hops and notifies
+ *     anomalies when exceeds threshold.
+ * 15) Rule interface-in-errors-iagent.rule, monitors interface input errors 
+ *     and notifies anomalies when exceeds threshold.
+ * 16) Rule interface-out-errors-iagent.rule, monitors interface output errors
+ *     and notifies anomalies when exceeds threshold.
+ * 17) Rule interface-in-traffic-iagent.rule, monitors interface input traffic
+ *     and notifies anomalies when exceeds threshold.
+ * 18) Rule interface-out-traffic-iagent.rule, monitors interface output traffic
+ *     and notifies anomalies when exceeds threshold. 
+ * 19) Rule interface-flaps-iagent.rule, monitors interface flaps and notifies
+ *     anomalies when exceeds threshold. 
+ * 20) Rule interface-status-iagent.rule, monitors interface status and notifies
+ *     anomalies when exceeds threshold. 
+ * 21) Rule bgp-neighbor-state-iagent.rule, monitors bgp state and notifies
+ *     anomalies 
+ * 22) Rule bgp-neighbor-flap-iagent.rule, monitors interface status and notifies
+ *     anomalies when exceeds threshold.
+ * 23) Rule vxlan-vport.rule, monitors vport count and notifies anomalies
+ *     when exceeds threshold.
+ * 24) Rule evpn-db-state-netconf.rule, Verify evpn database state has duplicate
+ *     mac entry, which is detected by same vlan and same Active-source
+ * 25) Rule verify-mac-ip-netconf.rule, collects dynamic MAC address using udf
+ *     and updates dependent rules sensor table
+ * 26) Rule check-remote-vtep-count-netconf.rule, Collects remote VTEP counts per
+ *     VLAN periodically and notifies less than expected
+ * 27) Rule evpn-vxlan-monitor-count-online-netconf.rule, Check NHCount on fpc
+ *     below threshold passed as argument, fpc is dynamically derived
+ * 28) Rule check-flabels-count-netconf.rule, collects flabels token count
+ *     periodically and notifies anomalies when flabels token count is above
+ *     static threshold
+ * 29) Rule check-mac-count-netconf.rule, monitors MAC count in ethernet switching
+ *     table periodically and notifies in case of anomalies
```

Comment:
s/and notifies in case of anomalies/and notifies anomaly when MAC count is above threshold/g

Reply:
updated the rule